lib/span-data.js

```diff
@@ -82,6 +92,23 @@ SpanData.prototype.close = function() {
 };
 
 /**
+ * Trace API expects stack frames to be a JSON string with the following
+ * structure:
+ * STACK_TRACE := { "stack_frame" : [ FRAMES ] }
+ * FRAMES := { "class_name" : CLASS_NAME, "file_name" : FILE_NAME,
+ *             "line_number" : LINE_NUMBER, "method_name" : METHOD_NAME }*
+ * @constructor @private
+ */
+function StackFrame(className, methodName, fileName, lineNumber, columnNumber) {
+  if (className)   { this.class_name = className;  }
```

Comment:
Why are these if's necessary? Does the visible behavior change if they are not included?